optaplanner-quickstarts/spring-boot-school-timetabling/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.optaplanner.quickstarts</groupId>
+    <artifactId>optaplanner-spring-boot-school-timetabling-quickstart</artifactId>
+    <version>7.38.0-SNAPSHOT</version>
```

Comment:
Needs to change during releases...

Reply:
Honestly, since it doesn't share our POM hierarchy, I wonder if it even belongs in this repo. It doesn't behave like all the other modules, so maybe it's not like the other modules.

Reply:
Yea, the downside is if we push it it's own repo, we get cross-repo PR's (think git actions) and find-in-path searches that might not include it (because deverlopers won't know to add that repo in their IDE too)
I think this is the lesser evil.

Reply:
Ah, an optaplanner-distribution would also need to move into it's own repo too: it depends on the quickstarts which depend on optaplanner starter/extension/core